SimConnectMSFS/WasmModuleUpdater.cs

```diff
@@ -155,22 +155,19 @@ public bool WasmModulesAreDifferent()
 
             string installedWASM;
             string mobiflightWASM;
-            string installedEvents;
-            string mobiflightEvents;
 
             if (CommunityFolder == null) return true;
 
-
-            if (!File.Exists(CommunityFolder + $@"\mobiflight-event-module\modules\{WasmModuleName}"))
+            string wasmModulePath = Path.Combine(CommunityFolder, @"mobiflight-event-module\modules\", WasmModuleName);
+            if (!File.Exists(wasmModulePath))
+            { 
                 return true;
+            }
 
             installedWASM = CalculateMD5(CommunityFolder + $@"\mobiflight-event-module\modules\{WasmModuleName}");
             mobiflightWASM = CalculateMD5($@".\MSFS2020-module\mobiflight-event-module\modules\{WasmModuleName}");
 
-            installedEvents = CalculateMD5(CommunityFolder + $@"\mobiflight-event-module\modules\{WasmEventsTxtFile}");
-            mobiflightEvents = CalculateMD5($@".\MSFS2020-module\mobiflight-event-module\modules\{WasmEventsTxtFile}");
```

Comment:
This was causing a crash locally because the file doesn't exist anymore in a normal install. Removed since it's no longer needed.